### DISCUSSION TITLE
Navigation: Cannot set Home icon/name

### DISCUSSION THREAD

**Discussion by q9f** [forem/forem#15920] [Feature Requests]
**Describe the bug**

Admins can not customize the "Home" item in the left navigation.

**To Reproduce**

![Screenshot 2021-12-30 at 12-32-22 State D](https://user-images.githubusercontent.com/58883403/147748408-dabce30b-6adc-402c-8c4c-d167a1f64b1c.png)

![Screenshot 2021-12-30 at 12-34-07 Navigation Links](https://user-images.githubusercontent.com/58883403/147748416-a608bead-d3ca-4bd0-b766-6da6d357575b.png)

**Expected behavior**

The Home item should be customizable as any other item.

**Additional context**

* Forem Cloud `72975ce8623e` instance
* related discussion https://forem.dev/afr/how-to-strip-out-all-emojis-4b7h
* similar issue #15722

**Comment by q9f** [comment]
My ultimate motivation is to get rid of all emojis without using any CSS hacks. I would also like to customize the icons of the User Settings navigation.

**Reply by Ricardo7495** [reply]
I thinks it is not possible. As I tried alot for [Carpet cleaners Clarksville TN](https://aactionsteamer.com/) to remove the emoji but all in vain. The CSS coding was the ultimate source for us to remove it from our carpet cleaning platform where we are guiding our users about the carpet cleaning and showing different methods of cleaning. While the emoji was not related to our topic. So we removed it through css.

**Comment by citizen428** [comment]
Thanks for the suggestion @q9f. Since this is essentially a feature request I moved it to a GH discussion so we can better gauge how many other people are interested in this.
